runtimes/spiritnet/src/lib.rs

```diff
@@ -916,7 +919,7 @@ construct_runtime! {
 		UncheckedExtrinsic = UncheckedExtrinsic,
 	{
 		System: frame_system = 0,
-		RandomnessCollectiveFlip: pallet_randomness_collective_flip = 1,
+		RandomnessCollectiveFlip: pallet_insecure_randomness_collective_flip = 1,
```

Comment:
Let's remove this please

Reply:
Same as above, plus a cleanup PR: https://github.com/KILTprotocol/kilt-node/pull/500/commits/ac9ab1fed82794c685a47a16e26ebd4ddbcc7002